parser_library/src/processing/instruction_sets/ca_processor.cpp

```diff
@@ -128,6 +128,9 @@ ca_processor::SET_info ca_processor::get_SET_symbol(const semantics::complete_st
     else
         set_sym = hlasm_ctx.template create_local_variable<T>(name, is_scalar_expression)->access_set_symbol_base();
```

Comment:
the return value may be nullptr now... needs to be tested before calling access_set_symbol_base()